input/docs/issue-providers/gitrepository/features.md

```diff
@@ -0,0 +1,36 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.GitRepository addin.
+---
+The [Cake.Issues.GitRepository addin] provides the following features.
+
+# Basic features
+
+* Checks if binary files are tracked by Git LFS.
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
+
+|                                                                    | Property                          | Remarks                        |
+|--------------------------------------------------------------------|-----------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderType`             |                                |
```

Comment:
why not linking the properties to [IIssue](https://cakebuild.net/api/Cake.Issues/IIssue/#Properties) docu?
If there are breaking changes in IIssue (renaming property) this would be detected regenerating this docu.

Reply:
These will currently be auto-linked (and won't break build if renamed)